includes/definitions/mrv-optidriver.yaml

```diff
@@ -0,0 +1,10 @@
+os: mrv-optidriver
+group: mrv
```

Comment:
Do you need this? It's designed to bring a number of OS' together so polling can be done centrally.